### PR TITLE
Fix function comments based on best practices from Effective Go

### PR DESCRIPTION
**Pull request by CodeLingoTeam**
Hi, we updated some exported function comments based on best practices from [Effective Go](https://golang.org/doc/effective_go.html#commentary). It’s admittedly a relatively minor fix up. Does this help you?

### CODE REVIEW

File: mgl32/shapes.go [hunk 1]
```diff
@@ -212,7 +212,7 @@ func BezierSurface(u, v float32, cPoints [][]Vec3) Vec3 {
 	return point
 }
 
-// Does interpolation over a spline of several bezier curves. Each bezier curve must have a finite range,
+// BezierSplineInterpolate2D Does interpolation over a spline of several bezier curves. Each bezier curve must have a finite range,
```
Comment (dmitshur) [inline mgl32/shapes.go]: Can you change "Does" to "does" here, and everywhere similar below?

File: mgl32/vector.go [hunk 2]
```diff
@@ -186,7 +186,7 @@ func (v Vec2) Y() float32 {
 	return v[1]
 }
 
-// Does the vector outer product
+// OuterProd2 Does the vector outer product
```
Comment (dmitshur) [inline mgl32/vector.go]: Here and below as well.

File: mgl32/vector.go [hunk 3]
```diff
@@ -347,7 +347,7 @@ func (v Vec3) Z() float32 {
 	return v[2]
 }
 
-// Does the vector outer product
+// OuterProd2 Does the vector outer product
```
Comment (dmitshur) [inline mgl32/vector.go]: Here and below, too.

**Comment by pwaller** [comment]
@dmitshur judging from the user's activity, it is a bot, so I think we'll have to make the fix ourselves.

@CodeLingoTeam @CodeLingoBot if you're listening, I think @dmitshur's suggestion would apply as a general rule, if the bot could be fixed.

Also interesting that it looks like our travis configuration is broken. I'll make a new PR and try and fix both.

**Comment by pwaller** [comment]
There is another problem with these changes, they're actually changing generated code. See the header:

```
// This file is generated by codegen.go; DO NOT EDIT
// Edit vector.tmpl and run "go generate" to make changes.
```

@CodeLingoTeam @CodeLingoBot as per https://golang.org/s/generatedcode - these files should not have been edited anyway. Closing this PR, I'll see about submitting fixes to the underlying templates.

**Comment by pwaller** [comment]
@CodeLingoTeam I have some feedback on your bot and this PR.

This did cost me an hour of work to fix in the end, and I guess it is fairly low value improvement.

It would be useful if you made it clear in the pull request that this is a bot making the pull request, and it would also have been useful if you had included a link to a way to reproduce your automated changes, since I ended up redoing them due to the templating involved.

**Pull request closed without merging**